lib/exavier/cover.ex

```diff
@@ -51,7 +52,9 @@ defmodule Exavier.Cover do
           {:error, :non_existing} ->
             Logger.error("Could not find module #{non_default_module_name} defined in option :test_files_to_modules for #{test_file}.")
 
-          {:ok, _} -> {:ok, :compiled}
+          {:ok, _} ->
+            source = non_default_module_name.__info__(:compile)[:source]
```

Comment:
Btw, using `Module.__info__` I think effectively allows me to drop the feature for config option [`:test_files_to_modules`](https://github.com/dnlserrano/exavier/blob/master/config/config.exs#L4-L9) (to override module name inference from test files, given with `<MODULE>.__info__(:module)` this should be straightforward. Right?

Reply:
I'm not sure. Because for using `__info__` or `module_info` you need an actual module, an atom. And `test_files_to_modules` describes actually a dependancy of the tests. We rely on conventions. But it could be any module name in any filename testing one or more modules. There is no any restrictions in the language. That's why I think this (or similar) configuration is unavoidable.

Reply:
Yeah, you're right. I misunderstood this.